eng/targets/Helix.props

```diff
@@ -12,7 +12,6 @@
   <PropertyGroup>
     <CreateHelixPayload>true</CreateHelixPayload>
     <HelixTimeout>00:30:00</HelixTimeout>
-    <HelixTimeout Condition="$(HelixTargetQueue.StartsWith('Windows.10.Amd64'))">00:40:00</HelixTimeout>
```

Comment:
Surprised this doesn't conflict w/ d4d4c4bda5f3. Even though GitHub thinks it's fine because we both deleted this line, probably need to rebase your branch and make sure the now-later line is deleted.

Reply:
OIC there's two copies of this line now. (Might be my bad, sorry.) Need to delete https://github.com/dotnet/aspnetcore/pull/38575/files#diff-ed02e4f0280f5e5a03fce7e0a5c047a2115f07f239dd16d8374025a16dea0e4bR29 too